include/orc/string_pool.hpp

```diff
@@ -16,39 +16,107 @@
 
 struct pool_string;
 
+/*
+    Add a string to the pool. This is not thread safe.
```

Comment:
Why is this operation not thread safe anymore?

Reply:
good catch, again! old comment.